src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,97 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+  padding-bottom: 150px;
+  padding-left: 75px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 78px;
+    padding-left: 0;
+    padding-top: 50px;
+  }
+}
+
+.allTextWrap {
+  grid-column-end: 11;
+  grid-column-start: 2;
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-row: 2;
+}
```

Comment:
Just a general comment about your grid. The `allTextWrap` element that you've introduced is causing things to be a bit awkward. You're currently halfway between two worlds, and you need to pick one, but what you have now is partially doing things one way, partially doing things another, and you have various CSS properties that have no effect because of the extra layer of indirection.

You can only set the grid child properties on the direct children of a grid parent element. In your current code, `.gridParent` is the grid parent, since it has `display: grid` set, and `.allTextWrap` is the one and only grid child. `.title` and `.gridAreaBottom` are children of `.allTextWrap`, which means that they are not grid children (they are grandchildren) and therefore none of the grid properties will do anything on them.

To be very specific, setting `grid-row: 2` will do nothing on `.gridAreaBottom` doesn't do anything because, despite its name, it is not a grid child.

There are two different approaches you can take:

1. You get rid of `.allTextWrap` altogether, both in the HTML and in CSS, and you have `.title` and `.gridAreaBottom` specify their grid row and/or column.
2. You leave `.allTextWrap` in, but get rid of all the grid-related properties on `.title` and `.gridAreaBottom`.

I'd probably prefer the former, since it reduces the number of elements in the component, which simplifies things.

Reply:
👍 Deleted allTextWrap. Gave title and gridAreaBottom own grid column start and end.